docs/source/item-validation.rst

```diff
@@ -87,6 +87,34 @@ an example of a schema for the quotes item from the :doc:`tutorial </getting-sta
     ]
   }
 
+With Cerberus
+-------------
+
+`Cerberus`_ is a powerful yet simple and lightweight data validation
+tool, designed to be ​extensible​, allowing for custom validation​ and has ​no
+dependencies. You can define what the field contains, what is required, the type of
+each field, as well as dependencies and regex.
+
+.. warning::
+
+   You need to install `cerberus`_ to use this feature.
+
+This `usage`_ and `validation-rules`_ guide explain the main keywords and how to make a
+schema. Here we have an example of a schema for the quotes item from the
+:doc:`tutorial </getting-started>`.
+
+.. code-block:: json
+
+    {
+        "quote": {"type": "string", "required": true},
+        "author": {"type": "string", "required": true},
+        "author_url": {"type": "string"},
+        "tags": {"type": "list"}
+    }
+
+To use Cerberus validation, you would need to add
+:ref:`SPIDERMON_VALIDATION_CERBERUS` setting to your `settings.py`
```

Comment:
I would not mention `settings.py` directly, as Scrapy provides several different places where settings may be defined.

You could say ‘to your Scrapy settings’, and link ‘Scrapy settings’ to https://docs.scrapy.org/en/latest/topics/settings.html

It may be a good opportunity to add [InterSphinx](https://www.sphinx-doc.org/en/master/usage/extensions/intersphinx.html) to the development dependencies and documentation configuration, and use it here to link to the Scrapy documentation.

Reply:
Great suggestion, implemented. 

+1, I wanted to have a discussion on adding InterSphinx. I think it would be quite powerful and helps link up external docs better. We can also add up some mentions of Spidermon in the Scrapy docs as well to keep projects connected.

Reply:
Let me know if we like to go ahead with InterSphinx.